app/src/main/java/com/mealsmadeeasy/ui/login/LoginActivity.kt

```diff
@@ -0,0 +1,92 @@
+package com.mealsmadeeasy.ui.login
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.support.design.widget.Snackbar
+import com.google.android.gms.auth.api.signin.GoogleSignIn
+import com.google.android.gms.auth.api.signin.GoogleSignInAccount
+import com.google.android.gms.auth.api.signin.GoogleSignInClient
+import com.google.android.gms.auth.api.signin.GoogleSignInOptions
+import com.google.android.gms.common.SignInButton
+import com.google.android.gms.common.api.ApiException
+import com.google.firebase.auth.FirebaseAuth
+import com.google.firebase.auth.GoogleAuthProvider
+import com.mealsmadeeasy.R
+import com.mealsmadeeasy.ui.BaseActivity
+import com.mealsmadeeasy.ui.home.HomeActivity
+
+class LoginActivity : BaseActivity() {
+
+    private val RC_SIGN_IN = 9001
```

Comment:
Can we make this a `const val`? (It'll either need to be above the class header or in the companion object, whichever you prefer).